README.md

```diff
@@ -122,17 +122,22 @@ _Context: global_
 
 Use this command to run tests on a package. Currently, there are two types of tests available.
 
+_Context: package_
+
 #### Pipeline Tests
 
 These tests allow you to exercise any Ingest Node Pipelines defined by your packages.
 
+They require only an Elasticsearch node to be up and running, to use the [Simulate API](https://www.elastic.co/guide/en/elasticsearch/reference/master/simulate-pipeline-api.html).
```

Comment:
Nit: I would leave out this detail so it's consistent with the section right below on System Tests. Also, it's more of a "how do pipeline tests work" detail not so much "what are pipeline tests". I think the latter is sufficient for the overview README.

Reply:
Fixed